<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionEditor</class>
 <widget class="QDialog" name="ActionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Action Editor</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEditAnimationName">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidgetMain">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabD">
       <attribute name="title">
        <string>Actions</string>
       </attribute>
       <widget class="QListWidget" name="listAnimation">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>611</width>
          <height>361</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Curves</string>
       </attribute>
       <widget class="QListWidget" name="listCurves">
        <property name="geometry">
         <rect>
          <x>-5</x>
          <y>1</y>
          <width>621</width>
          <height>361</height>
         </rect>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Edit</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonDone">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>440</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>ActionEditor</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
